tests/registry_disk_test.go

```diff
@@ -168,4 +171,34 @@ var _ = Describe("ContainerDisk", func() {
 			})
 		})
 	})
+
+	Describe("Starting with virtio-win", func() {
+		Context("with virtio-win as secondary disk", func() {
+			It("should boot and have the virtio as sata CDROM", func() {
+				vmi := tests.NewRandomVMIWithEphemeralDisk(tests.ContainerDiskFor(tests.ContainerDiskAlpine))
+				tests.AddEphemeralCdrom(vmi, "disk4", "sata", fmt.Sprintf("%s/virtio-container-disk:%s", tests.KubeVirtRepoPrefix, tests.KubeVirtVersionTag))
+
+				By("Starting the VirtualMachineInstance")
+				obj, err := virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(tests.NamespaceTestDefault).Body(vmi).Do().Get()
+				Expect(err).To(BeNil(), "expected vmi to start with no problem")
+				tests.WaitForSuccessfulVMIStart(obj)
+
+				By("Checking whether the second disk really contains virtio drivers")
+				expecter, err := tests.LoggedInAlpineExpecter(vmi)
+				Expect(err).ToNot(HaveOccurred(), "expected alpine to login properly")
+				defer expecter.Close()
+
+				_, err := expecter.ExpectBatch([]expect.Batcher{
```

Comment:
Travis fails here
tests/registry_disk_test.go:191:12: no new variables on left side of :=